### ISSUE TITLE
Issue on add

### ISSUE THREAD

**Issue by NSmeef** [vibrato/awskeyring#7]
```
Traceback (most recent call last):
	11: from /usr/local/bin/awskeyring:23:in `<main>'
	10: from /usr/local/bin/awskeyring:23:in `load'
	 9: from /usr/local/lib/ruby/gems/2.5.0/gems/awskeyring-0.0.4/exe/awskeyring:10:in `<top (required)>'
	 8: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/base.rb:466:in `start'
	 7: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
	 6: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
	 5: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
	 4: from /usr/local/lib/ruby/gems/2.5.0/gems/awskeyring-0.0.4/lib/awskeyring_command.rb:95:in `add'
	 3: from /usr/local/lib/ruby/gems/2.5.0/gems/awskeyring-0.0.4/lib/awskeyring.rb:60:in `add_item'
	 2: from /usr/local/lib/ruby/gems/2.5.0/gems/awskeyring-0.0.4/lib/awskeyring.rb:56:in `all_items'
	 1: from /usr/local/lib/ruby/gems/2.5.0/gems/awskeyring-0.0.4/lib/awskeyring.rb:34:in `load_keychain'
/usr/local/lib/ruby/gems/2.5.0/gems/ruby-keychain-0.3.2/lib/keychain.rb:62:in `open': ArgumentError (ArgumentError)
```

**Comment by NSmeef** [comment]
On `awskeyring add [account]`

**Comment by tristanmorgan** [comment]
have you run `awskeyring initialise`? it should have returned a message first though....

**Comment by tristanmorgan** [comment]
Reopening because the check for missing config is the root cause of this.

**Comment by tristanmorgan** [comment]
resolved with #8
